packages/expo/src/launch/registerRootComponent.tsx

```diff
@@ -5,7 +5,7 @@ import withExpoRoot from './withExpoRoot';
 import { InitialProps } from './withExpoRoot.types';
 
 export default function registerRootComponent<P extends InitialProps>(
-  component: React.ComponentClass<P>
+  component: React.ComponentType<P>
 ): void {
   // @ts-ignore: TypeScript says ComponentClass<P> does not satisfy ComponentClass<any>
```

Comment:
Please update these comments too, if the ts-ignore is still needed

Reply:
good call, it's not needed 🙂